Source/Scenes/Main/UnlockAppWithPincode/UnlockAppWithPincodeViewModel.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  UnlockAppWithPincodeViewModel.swift
+//  Zupreme
+//
+//  Created by Alexander Cyon on 2018-11-13.
+//  Copyright © 2018 Open Zesame. All rights reserved.
+//
+
+import Foundation
+import RxCocoa
+import RxSwift
+
+private typealias € = L10n.Scene.UnlockAppWithPincode.Title
+
+enum UnlockAppWithPincodeNavigation: TrackedUserAction {
+    case userFinished
```

Comment:
@andiradulescu Yeah vague because it needs to cover the three different scenarios:
1.  User aborted removal of pincode
2.  User entered the correct pincode so that she can remove her pincode
3.  User did unlock app

All three of which same thing happens in the coordinator. Thus it is actually not three different navigation outcomes FROM the ViewModels perspective. BUT yeah, it should be the `ManagePincodeCoordinator` that makes such a decision. So it would be more correct to have these three different cases in the enum but letting them result in the same code being executed by the coordinator.

So I will change it back to being three :) thx!